packages/frontend/src/components/AppConnectionRow/index.tsx

```diff
@@ -34,6 +35,7 @@ const countTranslation = (value: React.ReactNode) => (
 
 function AppConnectionRow(props: AppConnectionRowProps): React.ReactElement {
   const toast = useToast()
+  const [dialogType, setDialogType] = useState<AlertDialogType>('delete') // delete by default
```

Comment:
i think you won't need this since there's only one `delete` action possible in this file right?

Reply:
good point, unlikely to have other actions any time soon, will remove